www/artworks/post.php

```diff
@@ -9,7 +9,28 @@
 
 session_start();
 
+function post_max_size_bytes(): int{
```

Comment:
sorry, another nit: Consider moving this function to the top of the file under the `require_once` line.